happybase/counter_batch.py

```diff
@@ -0,0 +1,48 @@
+from happybase.hbase.ttypes import TIncrement
+from collections import defaultdict
+
+
+class CounterBatch(object):
+    def __init__(self, table, batch_size=None):
+        self.table = table
+        self.batch_size = batch_size
+        self.batch = defaultdict(int)
+        self.batch_count = 0
+
+    def counter_inc(self, row, column, value=1):
+        self.batch[(row, column)] += value
+        self.batch_count += 1
```

Comment:
i think this is incorrect. if the `(row, column)` tuple was already present this count will be wrong. i guess `len(self.batch)` is a better count ;)